AnisimovAN/Lab1/Lab1/alfa.cpp

```diff
@@ -12,32 +12,31 @@ int main()
 	cout << "Insert second argument" << endl;
 	cin >> right;
 
+
 	switch (sign)
 	{
-	case '+': result = left + right;
+	case '+':  cout << left << sign << right << " = " << left + right;

```

Comment:
Now it's totally incorrect. You have to store the result in variable and print in after all checks are done.